lib/rgit/cli.rb

```diff
@@ -1,31 +1,234 @@
-
+require 'byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.